site-graph/http/middleware.go

```diff
@@ -0,0 +1,28 @@
+package http
+
+import (
+	"net/http"
+)
+
+// IsAuth return false if username or password
+// is not allowed to access application.
+func IsAuth(username, password string) bool {
+	return username == "gobridge" && password == "secret"
```

Comment:
timing attack vulnerable
to fix - see https://golang.org/pkg/crypto/subtle/#ConstantTimeCompare